test/quantization/core/test_quantized_op.py

```diff
@@ -3535,12 +3535,14 @@ class TestQuantizedLinear(TestCase):
            use_bias=st.booleans(),
            use_relu=st.booleans(),
            use_multi_dim_input=st.booleans(),
-           use_channelwise=st.booleans())
+           use_channelwise=st.booleans(),
+           use_fused_qdq_skip_quant=st.booleans())
```

Comment:
this test case is getting really confusing with so many flags, should we just test this separately?  

Lines such as 

```
            if use_fused_qdq_skip_quant:
                Y_q_dq = qlinear(X, X_scale, X_zp, W_prepack)
            else:
                Y_q = qlinear(X_q, W_prepack, Y_scale, Y_zp)
```

are pretty hard to read for someone who wasn't on this PR review

Reply:
yeah makes sense, although this test also is extremely long I feel we shouldn't just copy paste code, how about making a helper function and expose some branches as argument?

Reply:
actually, are you suggesting to copy-paste the code? I don't see other ways to remove this `if` check..

Reply:
I don't feel too strongly on how.  If it was me, I'd probably copy-pasta, as I rank readability significantly higher than code reuse in this context.  I guess you could also refactor it somehow, or just ignore the feedback and leave as is.  Either way lgtm.